tests/e2e/cht/pregnancy.spec.js

```diff
@@ -0,0 +1,177 @@
+const utils = require('../../utils');
+const commonElements = require('../../page-objects/common/common.po');
+const loginPage = require('../../page-objects/login/login.po');
+const contactsPage = require('../../page-objects/contacts/contacts.po');
+const helper = require('../../helper');
+// const pregnancyFormPo = require('../../page-objects/forms/cht/pregnancy-form.po');
+const analyticsPo = require('../../page-objects/analytics/analytics.po');
+// const pregnancyHomeVisit = require('../../page-objects/forms/cht/pregnancy-home-visit.po'); 
+const moment = require('moment');
+// const deliverPo = require('../../page-objects/forms/cht/delivery.po'); 
+const Factory = require('rosie').Factory;
+require('/home/newt/dev/cht-core/tests/factories/reports/cht/pregnancy.js');
+const formFiller = require('../../form-filling').fillForm;
+require('../../factories/reports/cht/pregnancy-visit');
+require('../../factories/reports/cht/delivery');
+const genericFormPo = require('../../page-objects/forms/generic-form.po');
+require('/home/newt/dev/cht-core/tests/factories/reports/cht/pregnancy.js');
+
+const password = 'Secret_1';
+const district = {
```

Comment:
This is still a TODO for me. I want to pull the hierarchy into a facotry for use but wanted to get the form filling part into a draft review.

Reply:
I actually put what I think is a better option in the Pregnancy factory.